src/frontend/js/order.js

```diff
@@ -3,6 +3,35 @@ $(function() {
     // Javascript of the order application.
     // **************************************
 
+    if ($('.ui.order-override.modal')) {
+      $('.ui.order-override.modal').modal('show');
```

Comment:
@aaboffill I agree, especially because this code is executed automatically after the page is ready. 

@peakwinter We may need to develop a way of managing these frontend codes, as eventually they're put and minified together. What do you suggest as the best practice on this? An app-based class/ID indicator on `<html>` element? We may also create an issue on this.